app/config/index.js

```diff
@@ -25,13 +26,13 @@ const config = {
   robotSheetName: process.env.ROBOT_SHEET_NAME,
   osPlacesApi: {
     baseUrl: process.env.OS_PLACES_API_BASE_URL,
-    token: process.env.OS_PLACES_API_KEY
+    token: getEnvironmentVariable('OS_PLACES_API_KEY')
```

Comment:
If you're changing the calls to use your new function, should you not make all values use this function for consistency?

Reply:
we can do, I've been doing it on an as needed basis at the moment.  It will make testing more difficult otherwise.